libs/ui-toolkit/src/components/async-renderer/async-renderer.tsx

```diff
@@ -15,15 +16,15 @@ export function AsyncRenderer<T = object>({
   children,
 }: AsyncRendererProps<T>) {
   if (error) {
-    return <Splash>Something went wrong: {error.message}</Splash>;
+    return <Splash>{t(`Something went wrong: ${error.message}`)}</Splash>;
```

Comment:
**nitpick (non-blocking)**: Not hugely important but when `t` is doing more work this will likely break due to error.message being unpredictable